docs/src/routes/__layout-homepage.svelte

```diff
@@ -9,10 +9,12 @@
   import '@svelteness/kit-docs/client/styles/theme.css'
   import '$lib/styles/fonts.css'
   import '$lib/styles/kit-docs.css'
+  import '@docsearch/css'
+  import '@svelteness/kit-docs/client/styles/docsearch.css'
```

Comment:
@Adamj1232 Can we remove this too? It's all related to search, but we don't use it in this layout.

Reply:
```suggestion
```